routes/api.js

```diff
@@ -190,14 +190,16 @@ router.post('/user-access', function(req, res, next) {
       };
     };
     db.grantPrivileges(res.locals.obj.userInCache['badge'], req.headers['station-id']).then(function(approval) {
-      cache.delete(req.headers['station-id']);
-      stationLog.addUser(res.locals.obj.userInCache['user'], res.locals.obj.userInCache['badge'], req.headers['station-id']);
-      res.set({
-        'Content-Type': 'text/plain',
-        'station-state': 'enabled',
-        'User-ID-String': res.locals.obj.userInCache['user']
-      })
-      return res.sendStatus(200);
+      db.logEvent('Granted permission to log into station', + req.headers['station-id'] + 'by' + res.locals.obj.user + 'for' + res.locals.obj.userInCache['badge']).then(function(){
+        cache.delete(req.headers['station-id']);
+        stationLog.addUser(res.locals.obj.userInCache['user'], res.locals.obj.userInCache['badge'], req.headers['station-id']);
```

Comment:
732	Granted permission to log into station	6byY,Xfor111	2018-03-01 20:18:48 +0000

This is what is recorded in the DB log after hitting this log event.  It doesn't look like res.locals.obj.user is working as you think it should be.  Also as you can tell by the 6byY,Xfor111 you need to edit this log to include proper spacing between words.  It should look something like this: 6 by X,Y, for 111